### PR TITLE
Fixes audio in poster page

### PR DESCRIPTION
**Pull request by mike-lvov**
Fixes audio absence in Poster page venue

### CODE REVIEW

File: src/hooks/twilio.ts [hunk 1]
```diff
@@ -226,14 +232,15 @@ export const useVideoRoomState = ({
     // https://media.twiliocdn.com/sdk/js/video/releases/2.7.1/docs/global.html#ConnectOptions
     connect(token, {
       name: roomName,
-      video: hasVideo,
+      video: isActiveParticipant,
+      audio: isActiveParticipant,
```
Comment (mike-lvov) [inline src/hooks/twilio.ts:236]: [NOTE]: The actual fix is in this line

**Review by yurii-lubynets — APPROVED** [review]

File: src/hooks/twilio.ts [hunk 2]
```diff
@@ -157,13 +163,13 @@ export const useParticipantState = ({
 export interface UseVideoRoomStateProps {
   userId?: string;
   roomName?: string;
-  showVideoByDefault?: boolean;
+  defaultActiveParticipant?: boolean;
```
Comment (0xdevalias) [inline src/hooks/twilio.ts:166]: Can we word this in a way that flows better like it used to? show

**Review by 0xdevalias — APPROVED** [review]
[CRO] Fix seems logical, renames seem pretty useful, minor question about the external API for defaults on the hook.

**Review by sunny-viktoryia — APPROVED** [review]
Looks good. :)

File: src/hooks/twilio.ts (hunk 2 above)
Comment (mike-lvov) [reply src/hooks/twilio.ts]: Yep, I tried to follow the naming patterns set in `useParticipantState`.

image

Renamed in https://github.com/sparkletown/sparkle/pull/1513/commits/db7071fb8e926299f9a82a9eface65db11ab5c1b

**Pull request merged**